### ISSUE TITLE
Handle multiple students

### ISSUE THREAD

**Issue by sid-kap** [yourPISD/yourpisd#11]
`Parser.parseStudents` currently only handles the case where there is one student. (The multiple-students case occurs with parent accounts that have multiple students.)

**Comment by theKidOfArcrania** [comment]
Yeah I think it handles the first student, we haven't gotten to implementing that yet though.

**Comment by theKidOfArcrania** [comment]
At least for the beta version, I'm probably going to just disable parent viewing (for now) so that we can push the app ASAP

**Comment by sid-kap** [comment]
That make sense---it's better to have some of it working than nothing working
